engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala

```diff
@@ -0,0 +1,37 @@
+package cromwell.services
+
+import akka.actor.Actor
+import com.typesafe.config.Config
+import cromwell.engine.db.DataAccess
+import cromwell.services.MetadataServiceActor._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.language.postfixOps
+import scala.util.{Failure, Success}
+
+// TODO: PBE: Will not be MetadataServiceActor until circular dependencies fixed.
```

Comment:
You might have done this already (in which case, disregard) but can you open a ticket noting that this will eventually need to move, what's blocking that from happening, etc - just so it doesn't get lost in the shuffle?